lib/api.js

```diff
@@ -7,9 +7,11 @@ const Workflow = require('./workflow/client');
 
 const errorHandler = require('./error-handler');
 
+const redisClient = require('redis').createClient();
```

Comment:
We will need to pass the connection options from config in here because deployed instances will not be on `localhost`.